docs/getting_started.rst

```diff
@@ -4,397 +4,4 @@
 Getting Started
```

Comment:
I am currently leaving this page to exist as a link to quickstart, in case there are previous links that exist directing people to this page, it won't be a page that doesn't exist.